app/templates/my-sessions/view.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="ui container">
+  <div class="ui row">
+    <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Edit Session Proposal'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+    <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Edit Speaker Details'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+    <button class="ui red button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Withdraw Proposal'}}</button>
+    {{#if device.isMobile}}
+      <div class="ui hidden fitted divider"></div>
+    {{/if}}
+  </div>
+  <div class="ui row">
+    <h2 class="ui header left aligned">
+      {{t 'Session Details'}}
+    </h2>
+    <div class="ui segment">
+      <h3 class="ui header">
+        <div>
+          {{#if (eq model.state 'accepted')}}
+            <div class="ui green large label">{{t 'Accepted'}}</div>
+          {{else if (eq model.state 'pending')}}
+            <div class="ui yellow large label">{{t 'Pending'}}</div>
+          {{else if (eq model.state 'rejected')}}
+            <div class="ui red large label">{{t 'Rejected'}}</div>
+          {{/if}}
+        </div>
+      </h3>
+      <div class="field">
+        <div class="ui divider"></div>
+        <h1 class="ui blue header"> {{model.title}} </h1>
+        <div class="ui header"> {{model.speaker.name}} <br> {{t 'From '}}{{moment-format model.startAt 'ddd, MMM DD HH:mm A'}}{{t ' to '}}{{moment-format model.endsAt 'ddd, MMM DD HH:mm A'}} </div>
+      </div>
+      <div class="ui divider"></div>
+      <h3 class="ui left aligned header">Short Abstract</h3>
+      <p> {{model.shortAbstract}} </p>
+      <h3 class="ui left aligned header">Session Type</h3>
+      <p>{{model.sessionType.name}}</p>
+      <h3 class="ui left aligned header">Track</h3>
+      <p>{{model.track.name}}</p>
+      <h3 class="ui left aligned header">Slide</h3>
+      <p>{{model.slidesUrl}}</p>
```

Comment:
@ritikamotwani Don't display the URL directly, follow the implementation as in open-event-legacy.